Decred Wallet/Extensions/UIButton.swift

```diff
@@ -54,4 +53,22 @@ extension UIButton {
             self.setNeedsLayout()
         }
     }
+
+    private func imageWithColor(color: UIColor) -> UIImage? {
```

Comment:
Please attend to this previous comment on this line. https://github.com/raedahgroup/dcrios/pull/469/files#r294531264